### ISSUE TITLE
Highlight a work on your user profile accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3988]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

 **Elements must have sufficient color contrast**
![Screen Shot 2019-09-12 at 8 34 35 PM](https://user-images.githubusercontent.com/9621070/64835914-c77d4180-d59c-11e9-8a92-48069af40006.png)

**Links must have discernible text**
![Screen Shot 2019-09-12 at 8 35 27 PM](https://user-images.githubusercontent.com/9621070/64835943-e085f280-d59c-11e9-9445-6e51e3154cbb.png)

![Screen Shot 2019-09-12 at 8 36 04 PM](https://user-images.githubusercontent.com/9621070/64835960-f5fb1c80-d59c-11e9-8967-1166f6d87e11.png)

![Screen Shot 2019-09-12 at 8 36 25 PM](https://user-images.githubusercontent.com/9621070/64835976-03180b80-d59d-11e9-96a1-316980fb5a53.png)

![Screen Shot 2019-09-12 at 8 36 44 PM](https://user-images.githubusercontent.com/9621070/64835984-0dd2a080-d59d-11e9-9e04-0fb411bfc4d9.png)

![Screen Shot 2019-09-12 at 8 37 04 PM](https://user-images.githubusercontent.com/9621070/64835996-19be6280-d59d-11e9-855a-adb7bdfa4ba6.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open Profile page in Nurax 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Video of Tab Test 

https://cl.ly/96b3265fc442/download/Screen%252520Recording%2525202019-09-12%252520at%25252008.38%252520PM.mov

Unable to tab through any of the links and buttons on the Profile page in Firefox and Safari. Chrome working fine.
